telluric/georaster.py

```diff
@@ -200,9 +206,9 @@ def _prepare_other_raster(one, other):
     if not (one.crs == other.crs and one.affine.almost_equals(other.affine) and one.shape == other.shape):
         if one.footprint().intersects(other.footprint()):
             other = other.crop(one.footprint(), resolution=one.resolution())
-            other = other.reproject(new_width=one.width, new_height=one.height,
-                                    dest_affine=one.affine, dst_crs=one.crs,
-                                    resampling=Resampling.nearest)
+            other = other._reproject(new_width=one.width, new_height=one.height,
```

Comment:
Similar concern here. I guess having the output parameters make `GeoRaster2._reproject` more efficient than `.reproject`? Perhaps it should be a public method with a different name, in a similar fashion than `crop` and `pixel_crop`.

Reply:
I am not sure, if I understand correctly your comment. What are the output parameters of `GeoRaster2._reproject` which we don't have in `GeoRaster2.reproject`?

Reply:
Sorry, badly written comment. In other words: if we are using a private method from the outside, it's a sign it shouldn't be private, no? If there's something preventing us from using `.reproject` directly (for example: that we already have all the input parameters that we need) perhaps we can find a public name for `_reproject`. But I can't think of any good names, so we might as well leave everything as it stands in this PR now.